<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>364</width>
    <height>477</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>60</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>First Name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="f_name_label">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>60</y>
      <width>113</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="l_name_label">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>120</y>
      <width>113</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>120</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Last Name</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="male_radio">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>190</y>
      <width>110</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Male</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="female_radio">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>190</y>
      <width>110</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Female</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="job_title_label">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>250</y>
      <width>171</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>250</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Job Titile</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>310</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Department</string>
    </property>
   </widget>
   <widget class="QPushButton" name="registration_btn">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>380</y>
      <width>88</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Register</string>
    </property>
   </widget>
   <widget class="QComboBox" name="dpt_combobox">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>310</y>
      <width>131</width>
      <height>27</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>--Select--</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Finance</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Human Resource</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ICT</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Information Security</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Internal Audit</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>management</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Administration</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>364</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>registration_btn</sender>
   <signal>clicked(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>233</x>
     <y>434</y>
    </hint>
    <hint type="destinationlabel">
     <x>181</x>
     <y>238</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
